src/test/ui/rfc-2632-const-trait-impl/assoc-type.stderr

```diff
@@ -1,19 +1,19 @@
-error[E0277]: cannot add `NonConstAdd` to `NonConstAdd`
+error[E0277]: the trait bound `NonConstAdd: ~const Add` is not satisfied
```

Comment:
The way to approach this might be adding a flag to the rustc_on_unimplemented directive that allows appending `in const contexts` instead of falling back to the generic message.

Reply:
Hmm.. I was hoping we could patch rustc_on_unimplemented to just append this, but I forgot that the messages are entirely custom and could make the wording weird or nonsensical.

So... a required field for rustc_on_unimplemented with an append snippet or just an entirely new message?